examples/ITK_Example17_MONAIWithPreregistration.ipynb

```diff
@@ -0,0 +1,502 @@
+      "text/plain": [
+       "<Figure size 1400x600 with 3 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Show examples of the pre-alignment\n",
+    "fixed, moving, aligned = dataset[0].values()\n",
+    "\n",
+    "fig = plt.figure(figsize=(14, 6))\n",
+    "plt.subplot(1, 3, 1)\n",
+    "plt.imshow(fixed.squeeze(), cmap='gray')\n",
+    "plt.title('fixed', fontsize=10)\n",
+    "plt.axis('off')\n",
+    "\n",
+    "plt.subplot(1, 3, 2)\n",
+    "plt.imshow(moving.squeeze(), cmap='gray')\n",
+    "plt.title('original moving', fontsize=10)\n",
+    "plt.axis('off')\n",
+    "\n",
+    "plt.subplot(1, 3, 3)\n",
+    "plt.imshow(aligned.squeeze(), cmap='gray')\n",
+    "plt.title('moving after elastix pre-alignment', fontsize=10)\n",
+    "plt.axis('off')\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "724c078c-f4ef-4f2a-9c30-82425b6f730f",
+   "metadata": {},
+   "source": [
+    "### Deep learning registration model"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "c8a5932c-4b91-4108-b394-85188dabe176",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "monai.networks.blocks.Warp: Using PyTorch native grid_sample.\n"
```

Comment:
Could linear interpolation be used? The "final registered" image looks rather ugly with those discontinuities?